content/05.limitations.md

```diff
@@ -0,0 +1,17 @@
+## Limitations of Study
+
+<!-- We suggest that you divide your limitations section into three steps: (1) identify the study limitations; (2) explain how they impact your study in detail; and (3) propose a direction for future studies and present alternatives.-->
+
+
+This study has some potential limitations.
+The brain tumor samples were collected over decades, RNA samples were prepared using more than one type of library method (stranded or poly-A, **Figure {@fig:S7}A**), and were sequenced at multiple centers.
+While we noted a strong library preparation batch effect (**Figure {@fig:S7}B**) and possible sequencing center batch effect (**Figure {@fig:S7}C**), we also found a very unbalanced distribution of cancer groups with stranded or poly-A RNA-Seq.
+We did not perform batch correction because removal of batch effects across unbalanced groups may induce false differences between groups in some cases [@doi:10.1093/biostatistics/kxv027; @doi:10.1016/j.tibtech.2017.02.012]. 
+Batch correction strategy is highly dependent on an analyst’s goals [@doi:10.1016/j.tibtech.2017.02.012], therefore, we made expression matrices available by library type and used only stranded expression matrices for transcriptomic analyses in (**Figure {@fig:4}**) and (**Figure {@fig:5}**).
+A second potential limitation is that performing analyses with all samples, rather than samples with high tumor purity might result in loss of information, such as subclonal variants or low-level oncogenic pathway expression.
+We observe potential cancer-specific correlations in tumor purity (e.g. lower purity in SEGA, PXA, teratoma (**Figure {@fig:S3}A**).
```

Comment:
sentence seems just plopped down here.. To me this sentence is a justification for why we used the median per cancer-group strategy for thresholding, but otherwise it's more of a result? So, I might move this after the next sentence and say we chose the median strategy due to imbalance.

Reply:
maybe i will move up to results section where I mention overall median purity